MessengerProj/src/main/res/values/strings.xml

```diff
@@ -304,107 +304,107 @@
     <string name="AccountNotConfigured">Account not configured</string>
-    <string name="AboutThisProgram">About Delta Chat</string>
+    <string name="AboutThisProgram">About</string>
     <string name="NotSet">Not set</string>
     <string name="NewChat">New chat</string>
     <string name="Deaddrop">Contact requests</string>
     <string name="DeaddropInChatlist">Show contact requests in chatlist</string>
     <string name="MuteAlways">Always muted</string>
-    <string name="AskStartChatWith">Start a chat with <![CDATA[<b>]]>%1$s<![CDATA[</b>]]>?</string>
-    <string name="DeaddropHint">To start a chat, click on the reply-arrows.</string>
-    <string name="NotYetImplemented">This function is not available or incomplete.</string>
-    <string name="DefaultStatusText">Sent with my Delta Chat Messenger: https://delta.chat</string>
+    <string name="AskStartChatWith">Chat with <![CDATA[<b>]]>%1$s<![CDATA[</b>]]>?</string>
+    <string name="DeaddropHint">Tap the reply-arrows to start chatting.</string>
+    <string name="NotYetImplemented">This function is either available or incomplete.</string>
+    <string name="DefaultStatusText">Corresponded under the aegis of Delta Chat Messenger: https://delta.chat</string>
     <string name="Name" >Name</string>
-    <string name="EmailAddress">Email address</string>
-    <string name="CannotDeleteContact">Cannot delete contacts in use, block contact instead.</string>
-    <string name="BadEmailAddress">Bad email address.</string>
+    <string name="EmailAddress">E-mail address</string>
+    <string name="CannotDeleteContact">Cannot delete contacts in use, block the contact instead.</string>
+    <string name="BadEmailAddress">Erroneous e-mail address.</string>
     <string name="ContactCreated">Contact created.</string>
     <string name="ViewProfile">View profile</string>
     <!-- Translators: please use a very short string here, it should not much longer than
     the string needed for video time+size (0:00, 12,3 Mib) -->
-    <string name="OneMoment">One moment …</string>
-    <string name="NoChatsHelp">Start messaging by pressing the new chat button in the bottom right corner or tap the menu button for more options.</string>
+    <string name="OneMoment">One moment…</string>
+    <string name="NoChatsHelp">Start messaging by pressing the \"New chat"\ button in the bottom right, or tap the \"Menu"\ button for more options.</string>
     <string name="Intro7Message"><![CDATA[<b>Delta Chat</b>]]> is safe for business use, compatible and standards-based.</string>
     <string name="InviteMenuEntry">Send invitations</string>
-    <string name="InviteText">I\'m using Delta Chat messenger now - %1$s - you can text me at %2$s</string>
+    <string name="InviteText">Shhh, Delta Chat messenger now - %1$s - you can text me at %2$s</string>
```

Comment:
Don't know - what does "Shhh" mean in this case?

Reply:
The sound you make to emphasise quietness.

Reply:
Ahhh, okay.

--
Shhh, I am not yet really awake.